python-openai/.env

```diff
@@ -0,0 +1 @@
+OPENAI_API_KEY=
```

Comment:
Not sure if we should include a `.env` file, because we encourage users to set secrets using the devbox/repo secrets instead 🤔

Reply:
Yeah I was not sure either but I wanted to include a solution enabling users to quickly get started. As CSB secrets are required to restart the VM it's not quick + it's not easy to point the users there.

Long term, that would be nice to have a tasks.json setting that the users can configure like on branch/fork we automatically show the env vars UI. Thinking about a setting like env var is required so segway to set it is mandatory.

Reply:
For now I will change the README for a CSB secret first approach.

Reply:
@CompuIves I changed the approach to encourage users to use CSB secrets